src/OrbitGl/TrackContainer.h

```diff
@@ -58,6 +58,7 @@ class TrackContainer final : public CaptureViewElement {
           iterator_timer_info,
       const absl::flat_hash_map<uint64_t, uint64_t>& iterator_id_to_function_id);
   void UpdateVerticalScroll(float ratio);
+  void ScrollVertically(int delta);
```

Comment:
I think those two function names are ambiguous. I'd suggest maybe renaming.
UpdateVerticalScroll -> SetVerticalScroll
ScrollVertically -> IncrementVerticalScroll

Reply:
It's a bit of a mess, because we also have Set and Get methods (used by CaptureWindow for Panning).

I'll update them to:
* UpdateVerticalScrollUsingRatio() ----> This method is used by scrollbars.
* IncrementVerticalScroll(float delta_y); ---> Using a delta_y that will be useful for pageUp and Down later